src/python/pants/core/goals/test.py

```diff
@@ -147,9 +147,9 @@ def level(self) -> LogLevel:
 
     def message(self) -> str:
         if self.skipped:
-            return f"{self.address} skipped."
+            return "skipped."
         status = "succeeded" if self.exit_code == 0 else f"failed (exit code {self.exit_code})"
-        message = f"{self.address} {status}."
+        message = f"{status}."
```

Comment:
The address (and environment, if set) is already a parameter, and so is rendered automatically here.